docs/process.md

```diff
@@ -118,3 +119,82 @@ that is:
 3. Go to the site repo, commit the changes, and push.
 
 [site_repo]: https://github.com/kripken/emscripten-site
+
+
+Packaging Emscripten
+--------------------
+
+If you package Emscripten for users in some manner, the above docs should be
```

Comment:
Does "above docs" just refer to the entirety of the emscripten.org web site? Is there a more specific section we could mention here?

Reply:
By "above" i meant "above in this document here". I'll clarify.